src/main/java/core/basesyntax/Draw.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Draw {
+    void toDrawFigure();
```

Comment:
The method name `toDrawFigure` could be improved for clarity and conciseness. A better name might be `draw` to directly imply the action to be performed.